src/App.js

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect } from 'react';
-import NotificationsPortal from '@redhat-cloud-services/frontend-components-notifications/cjs/NotificationPortal';
-import { Main } from '@redhat-cloud-services/frontend-components/components/cjs/Main';
+import NotificationsPortal from '@redhat-cloud-services/frontend-components-notifications/esm/NotificationPortal';
+import { Main } from '@redhat-cloud-services/frontend-components/components/esm/Main';
```

Comment:
You could instead upgrade to v3 FEC dependencies which resolve the asset type automatically.

Reply:
That's exactly what I am planning to do this in follow-up PRs. Now I aiming to switch to ESM, so I can update the wizard package.